tests/console/pcre.pm

```diff
@@ -24,7 +24,7 @@ sub run {
     assert_script_run "./test_pcrecpp";
     save_screenshot;
 
-    zypper_call("in php5");
+    zypper_call('in ' . (!leap_version_at_least('15') && !sle_version_at_least('15')) ? 'php7' : 'php5');
```

Comment:
If _php7_ is dropped on _TW_, _SLE >= 15_ and _Leap >=  15_

Then why is not:

```
(!leap_version_at_least('15') && !sle_version_at_least('15')) ? 'php5' : 'php7'
```

Reply:
php5 has been dropped from TW (not php7, that sticks around) - the same plan is for SLE (not sure of the 'when' though - but it is prepared in a staging already)

Reply:
ok, I typed it wrong.
But, the question is the same. I think the if sentence is reversed written.

Reply:
I really doubt `sle_version_at_least('15')` would work for TW in this case, it returned True if it's TW....

Reply:
@nilxam, are you answering to my question?

I don't mean reversed as

```
leap_version_at_least('15')

```
instead of 

```
!leap_version_at_least('15')
```

but as

```
? 'php5' : 'php7'
```

instead of

```
? 'php7' : 'php5'
```

Reply:
@SergioAtSUSE nah, my question is to @okurz

Reply:
my answer is as long as http://lord.arch/tests/7269/file/autoinst-log.txt shows the correct schedule I don't understand what you want to know

Reply:
Ok, need to rethink it

Reply:
Updated. I know made it very explicit by checking `is_leap` in combination with `leap_version_at_least` same as `is_sle` etc.

`is_leap_version_at_least(15)` might not work because there simply is no support for version 15 within that function but IMHO there is no sense in caring about that before we have any leap 15 tests so I consider myself done here.